parachain/pallets/ethereum-beacon-client/src/tests.rs

```diff
@@ -745,16 +823,16 @@ mod beacon_minimal_tests {
 			.expect("Time went backwards")
 			.as_secs();
 
-		let slot = initial_sync.header.slot;
-		let import_time = time_now + (slot * config::SECONDS_PER_SLOT); // Goerli genesis time + finalized header update time
+		let import_time = time_now + (update.finalized_header.slot * config::SECONDS_PER_SLOT); // Goerli genesis time + finalized header update time
 		let mock_pallet_time = import_time + 3600; // plus one hour
 
 		new_tester::<mock_minimal::Test>().execute_with(|| {
 			mock_minimal::Timestamp::set_timestamp(mock_pallet_time * 1000); // needs to be in milliseconds
 			LatestFinalizedHeaderState::<mock_minimal::Test>::set(FinalizedHeaderState {
 				beacon_block_root: Default::default(),
 				import_time,
-				beacon_slot: slot,
+				// use a child slot of the next updating to initialize
```

Comment:
I don't understand this comment?

Reply:
yes, comment here should be `parent` instead of `child`